kotlin/workflow-core/src/main/java/com/squareup/workflow/RenderContext.kt

```diff
@@ -86,32 +83,17 @@ interface RenderContext<StateT, in OutputT : Any> {
   ): ChildRenderingT
 
   /**
-   * Ensures [worker] is running. When the [Worker] emits an output or finishes, [handler] is called
-   * to determine the [WorkflowAction] to take.
+   * Ensures [worker] is running. When the [Worker] emits an output, [handler] is called
+   * to determine the [WorkflowAction] to take. When the worker finishes, nothing happens (although
+   * another render pass may be triggered).
```

Comment:
Maybe document how to use `transform` to react to finish if needed?

Reply:
Almost nobody cares, and I don't see the need to duplicate the Flow documentation.